android/build.gradle

```diff
@@ -67,4 +104,37 @@ dependencies {
 	playImplementation 'com.google.android.gms:play-services-auth:20.7.0'
 }
 
+def ndkPath = project.hasProperty('ndkPath') ? project.property('ndkPath') : System.getenv('ANDROID_SDK_ROOT')
```

Comment:
the `:ipn@aar` dependency above needs to be removed, it's being included in all variants